### DISCUSSION TITLE
Everytime sesh upgrades the session picker within tmux seems to stop working? Probably something I am doing wrong

### DISCUSSION THREAD

**Discussion by johnend** [joshmedeski/sesh#180] [Q&A] [answered]
So not really sure why this is the case, but for some reason every time I upgrade sesh I have to go in and reload/tweak my tmux config to make the picker work within tmux.

I am pretty much just using the example that's on the repo for using the fzf window and picker:

```shell
# session picker
bind    T   run-shell "sesh connect \"$(
            sesh list | fzf --tmux 55%,60% \
                --no-sort --ansi --border-label ' sesh ' --prompt '󱐋  ' \
                --header ' ^e everything ^m tmux ^g configs ^x zoxide ^d tmux kill ^f find' \
                --bind 'ctrl-e:change-prompt(  )+reload(sesh list)' \
                --bind 'ctrl-m:change-prompt(  )+reload(sesh list -t)' \
                --bind 'ctrl-x:change-prompt(󰥨  )+reload(sesh list -z)' \
                --bind 'ctrl-g:change-prompt(  )+reload(sesh list -c)' \
                --bind 'ctrl-f:change-prompt(  )+reload(fd -H -d 2 -t d -E .Trash . ~)' \
                --bind 'ctrl-d:change-prompt(󰧻  )+reload(tmux kill-session -t {})+change-prompt(󰧻   )+reload(sesh list)'
)\""

```

So not really sure what is happening, restarting the terminal, resourcing my config or shell doesn't seem to make it work again. The only thing that does is commenting out lines in the above until it does work, and then re-enabling them again.

Don't want to raise an issue though until I know for sure it's not something I have got set up wrong. 😄

**Comment by johnend** [comment]
Bit more context, when I hit enter on the session I wan to switch to, the list clears directories, and keeps tmux sessions. But does not switch to that session or close the fzf window.

This does work... sometimes, if the commenting/uncommenting allows it to progress.

**Comment by joshmedeski** [comment, marked as answer]
I've found that killing the tmux server helps sesh work better after an update.

Also, bugs are introduced sometimes when new features are introduced (ex: v2.1 broke something and was quickly fixed with v2.1.1). So keep an eye out for release notes because I'm still working on test coverage and stability.

Logging is going to be added very soon so it will be easier to debug this kind of stuff with a log file that can track behavior.
